tests/e2e/tests_runner.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir="$(dirname "$(readlink -f "$0")")"
+project_dir="$(readlink -f ${script_dir}/../..)"
+
+export GOPATH="$(mktemp -d)"
+tests_repo_dir="$GOPATH/src/github.com/kata-containers/tests"
+export CI=true
+
+# TODO: Debug should be enabled in order for some tests to enable the console
+# debug and search for patterns on agent logs. Probably console debug should
+# be enabled always.
+export DEBUG=true
+
+# TODO: some tests use the SKOPEO variable (which is a build time variable) to
+# decide whether run or not. This should be no longer the case when we replace
+# skopeo with image-rs.
+export SKOPEO=yes
+echo "IMPORTANT: Assume the image was built with SKOPEO=${SKOPEO}"
+
+# Tests will attempt to re-configure containerd. In our case it is already
+# proper set by the operator, so let's skip that step.
+export TESTS_CONFIGURE_CC_CONTAINERD=no
+
+# Test scripts rely on kata-runtime so it should be reacheable.
+export PATH="/opt/confidential-containers/bin:$PATH"
+
+clone_kata_tests() {
+	local cc_branch="CCv0"
+	#git clone --depth=1 --branch "$cc_branch" https://github.com/kata-containers/tests \
+	git clone https://github.com/kata-containers/tests \
+		"$tests_repo_dir"
+
+	# TODO: remove me.
```

Comment:
should this todo be fixed before the merge? does it depend on another pr?

Reply:
hi @jensfr ! thanks for the review!

Yes, I'd like to have everything fixed on kata's CCv0 branch before merge this PR. The good news is that there is only this https://github.com/kata-containers/tests/pull/5081 fix pending; once merged I will be able to switch out of my tree.

Reply:
done. On last update I removed that TODO, i.e., we are runnning the tests on CCv0 branch, not on my tree.